dev-requirements.txt

```diff
@@ -2,6 +2,9 @@
 # UNIT TESTING #
 ################
 
+# nox to run test sessions locally.
+nox==2021.10.1
+
```

Comment:
This could be moved out of the unit tests section, since its used for so much more than that. Also, the comment is irrelevant, they were only used to comment on things like why we install `mock` even tho it is a standard lib